molecule/default/molecule.yml

```diff
@@ -9,7 +9,15 @@ lint: |
     flake8
 platforms:
   - name: instance
-    image: centos:7
+    image: eniocarboni/docker-rockylinux-systemd:9
```

Comment:
For a role that simple, it might be sufficient to use a basic `rockylinux:9` image as you don't really need `systemd`

Reply:
Thank you @sbesson : the basic image was used and the test is now passing with your suggested change on https://github.com/pwalczysko/ansible-role-versioncontrol-utils/actions/runs/6548368078